lib/plugins/aws/lib/naming.js

```diff
@@ -638,4 +638,9 @@ module.exports = {
   getHttpApiLogGroupName() {
     return `/aws/http-api/${this.getStackName()}`;
   },
+  // Tags
+  getResourceTagNameRegex() {
```

Comment:
`naming.js` is probably not a good place as the intention behind this module is to be used when creating names for CloudFormation resources, I think we should go with separate module for such regexes - @medikoo, what do you think about this approach in general?

Reply:
Yeah - I looked through the code for other files where regular expressions where defined, but the only one I found that was actually defined in any sense (i.e. not just defined directly where it was used) was something about ALB naming.

A `regexp.js` on the same level as `naming.js` could do the trick. I'm sure there are more regular expressions that could use some basic tests.

Reply:
@Jalle19 great thanks for an attempt on solving this.

The way we handle schema configuration, is that regex should simply be put into schema (let's not mix into `naming`, as it serves very different purpose).

Concerning testing, we do not practice testing schema validation directly.

We test it indirectly by simply testing that defined properties produce desired effect.

In this case it'll be great to update (or add) a test which tests the property that relies on definition (e.g. `provider.stackTags`) and confirm that it accepts the tag with format that was previously rejected

Reply:
@medikoo so I should not introduce a new `regexp.js` file then?

Reply:
@Jall19 as this regex is used just in this single place, let's just put it inline there.

We do not practice secluding regular expression into some other dedicated file.

Reply:
There are quite a few complicated regular expressions used in the rule definitions that could use some rudimentary testing, especially for edge cases.